resources/accounts.md

```diff
@@ -0,0 +1,27 @@
+bash-5.0$ target/release/subkey generate --network litentry
```

Comment:
Better to have is off line
If it's public testnet, we should not do this
So eventually we need to figure out a way

Reply:
agree